src/tasks/batch_integration/api/task_info.yaml

```
@@ -4,6 +4,7 @@ v1:
   path: openproblems/tasks/batch_integration/README.md
   commit: 637163fba7d74ab5393c2adbee5354dcf4d46f85
 summary: Remove unwanted batch effects from scRNA data while retaining biologically meaningful variation.
+image: ["feature_thumbnail.svg", "embedding_thumbnail.svg", graph_thumbnail.svg]
```

Remove the other svgs and retain just the one.

```suggestion
image: thumbnail.svg
```